invokeai/app/api/dependencies.py

```diff
@@ -47,7 +47,7 @@ def initialize(args, config, event_handler_id: int):
         # TODO: Use a logger
         print(f">> Internet connectivity is {Globals.internet_available}")
 
-        generate = get_generate(args, config)
+        model_manager = get_model_manager(args, config)
```

Comment:
Can't this just construct a `ModelManager`? If not, why not?

Reply:
I'm following the pattern you used to create the `Generate` object. `get_model_manager()` is defined in `services.generate_initializer` and is 99% identical to the function originally named `get_generate()`. 

It's actually a fairly long initialization procedure that uses args to set up autoconversion, precision choice, and so forth. Since I'm not sure how the whole system's startup is being managed, I didn't make any changes here aside from swapping the creation of `Generate` with creation of a `ModelManager`

Should `get_model_manager()` be moved into `api.dependencies` ? This would make it easier to read the code.

Reply:
generate_initializer was "I know this is hacky, but I need to put this somewhere since so much has to be done to initialize the Generate class". I was hoping it'd be broken up as we refactored to more closely match the pattern for the other services.

Reply:
`generate_initializer` is now gone, but it is replaced by `model_manager_initializer`, which has about 120 lines of initialization code, about half of which is concerned with what to do if there is an exception when loading the default model. It has various args-processing code in it that belongs more in the front end than in the back end. Take a look and let me know where you think it should go.